test/e2e/e2e_test.go

```diff
@@ -0,0 +1,279 @@
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	servicecatalog "github.com/kubernetes-sigs/service-catalog/pkg/client/clientset_generated/clientset"
+	testutil "github.com/mongodb/mongodb-atlas-service-broker/test/util"
+	"github.com/stretchr/testify/assert"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	name              = "atlas-service-broker"
+	basicAuthUsername = "username"
+	basicAuthPassword = "password"
+)
+
+var (
+	kubeClient  *kubernetes.Clientset
+	svcatClient *servicecatalog.Clientset
+
+	atlasBaseURL    string
+	atlasGroupID    string
+	atlasPublicKey  string
+	atlasPrivateKey string
+	image           string
```

Comment:
Can we assign these as constants and call `testutil.GetEnvOrPanic` outside of the `TestMain`?

Reply:
That is absolutely brilliant, done!